### PR TITLE
docs: Add TabBar docs

### PR DESCRIPTION
**Pull request by kazo0**
What kind of change does this PR introduce?

- Documentation content changes

### CODE REVIEW

File: doc/controls/TabBar.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+![](../assets/tabbar-android-material-bottom.png)
+![](../assets/tabbar-ios-material-bottom.png)
+
+```xml
+xmlns:utu="using:Uno.Toolkit.UI"
+...
+
+<utu:TabBar Style="{StaticResource BottomTabBarStyle}">
+	<utu:TabBar.Items>
+		<utu:TabBarItem Content="Home">
```
Comment (Xiaoy312) [inline doc/controls/TabBar.md:101]: for the TabBar example, maybe use simpler `TabBarItem` like one-liner with just `Content` as text to get the idea across: you can nest `TabBarItem`s
and then add a few more example separately for `TabBarItem`, to show case what else is possible: icon and all IconElement variants, flyout...
or minimally, keep the flyout one last

File: doc/controls/TabBar.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    // Gets the newly selected TabBarItem
+    object? NewItem { get; }
+
+    // Gets the previously selected TabBarItem
+    object? OldItem { get; }
+}
+```
+
+## Styling `TabBar` & `TabBarItem`
+The default styles for `TabBar` and `TabBarItem` are very basic and it is assumed that they will have their styles set to either:
```
[the remark this answers is not in the document]
Comment (kazo0) [reply doc/controls/TabBar.md:54]: Yeah, I guess technically the default style would be considered the Fluent style?

I'm not even sure if the default Fluent style actually works. But things like TopAppBar or BottomAppBar are specifically for Material/Cupertino

I can change it to "recommended"

File: doc/controls-styles.md [hunk 3]
```diff
@@ -5,8 +5,8 @@ The `Uno.Toolkit.UI` library adds the follows controls:
 - [`Chip` and `ChipGroup`](controls\Chip.md): \[Material control\] Chips are compact elements that represent an input, attribute, or action.
 - `Divider`: \[Material control\] A divider is a thin line that groups content in lists and layouts.
 - [`DrawerControl`](controls\DrawerControl.md): A container to display additional content, in a hidden pane that can be revealed using swipe gesture, like a drawer.
+- [`TabBar`](controls\TabBar.md): A list selectable items that can be used to facilitate lateral navigation within an application.
```
Comment (agneszitte-nventive) [inline doc/controls-styles.md:8]:
```suggestion
- [`TabBar`](controls\TabBar.md): A list of selectable items that can be used to facilitate lateral navigation within an application.
```

File: doc/controls/TabBar.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
+# TabBar
+
+## Summary
+
+Represents a control that provides a list of `TabItem`s to select from. The `TabBar` selection can be used to trigger frame navigation or to toggle the visibility of views.
+
+## TabBarItem
+`TabBarItem` is a specialized `SelectorItem` that includes functionality such as triggering of an `ICommand` on click/tap or displaying a flyout.
```
Comment (agneszitte-nventive) [inline doc/controls/TabBar.md:8]:
```suggestion
`TabBarItem` is a specialized `SelectorItem` that includes functionality such as triggering an `ICommand` on click/tap or displaying a flyout.
```

File: doc/controls/TabBar.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+## Styling `TabBar` & `TabBarItem`
+Toolkit provides a barebones default style for `TabBar` and `TabBarItem`. It is recommended to use either:
+* [One of the pre-built styles](../controls-styles.md#control-styles) that come packaged within the `Uno.Toolkit.UI.Material` or `Uno.Toolkit.UI.Cupertino` libraries
+* A custom built style that is defined within the consuming application
+
+The styles that exist within the Toolkit Theming Libraries are built for the most common scenarios in which a `TabBar` could be used.
+
+### "Top" `TabBar` Style
+
+A common use for a `TabBar` would be to use it as a means of navigation between two or more views/pages that are all at the same level of hierarchy (Lateral Navigation). If the goal is provide lateral navigation for destinations that are at **any** level of hierachy, a `TabBar` styled as a `TopTabBar` is most appropriate.
```
Comment (agneszitte-nventive) [inline doc/controls/TabBar.md:61]:
```suggestion
A common use for a `TabBar` would be to use it as a means of navigation between two or more views/pages that are all at the same level of hierarchy (Lateral Navigation). If the goal is to provide lateral navigation for destinations that are at **any** level of the hierarchy, a `TabBar` styled as a `TopTabBar` is most appropriate. 
```

**Review by agneszitte-nventive — APPROVED** [review]

File: doc/controls/TabBar.md [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+Represents a control that provides a list of `TabItem`s to select from. The `TabBar` selection can be used to trigger frame navigation or to toggle the visibility of views.
+
+## TabBarItem
+`TabBarItem` is a specialized `SelectorItem` that includes functionality such as triggering an `ICommand` on click/tap or displaying a flyout.
+
+### Properties
+Properties|Type|Description
+-|-|-
+Command|ICommand|Gets or sets the command to invoke when the `TabBarItem` is pressed.
+CommandParameter|object|Gets or sets the parameter to pass to the Command property.
```
Comment (Xiaoy312) [inline doc/controls/TabBar.md:14]:
```suggestion
CommandParameter|object|Gets or sets the parameter to pass to the `Command` property.
```

**Review by Xiaoy312 — APPROVED** [review]

**Pull request merged**